runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -141,8 +156,10 @@ public void invoke() {
                     throw new IllegalStateException("Orchestrator can not be selected");
                 }
 
-            } catch (NetworkException | TimeoutException | IllegalStateException e) {
-                log.warn("WorkflowRequest: Error while running {} on attempt {}, cause:", this, x, e);
+            } catch (TimeoutException e) {
+                log.warn("WorkflowRequest: Error while running {} on attempt {}, cause:", this, retryIteration, e);
```

Comment:
another curly bracket required for exception

Reply:
Unfortunately, you are not correct, another bracket is not needed

Reply:
@pankti-m the `log.warn` is overloaded with `void warn(String var1, Object var2, Object var3);`  , so it doesn't need a string specific (i.e. '{}') to log the exception. You can just do `log.warn("msg", new ExceptionType());` and it will log the exception properly.

Reply:
Good to know.  Thanks @Maithem